### ISSUE TITLE
Multi-ID support

### ISSUE THREAD

**Issue by johnwalking** [RedAIGC/StoryMaker#30]
This is great! I would like to ask if it also supports expansion to three or four people?

**Comment by RedAIGC** [comment]
> This is great! I would like to ask if it also supports expansion to three or four people?

Thank you for your interest in our work. Currently, it can only support 1-2 people because the training set does not collect pictures of more people.

**Comment by gaoyixuan111** [comment]
> > 太棒了！请问还支持扩充到三人或四人吗？
> 
> 感谢您对我们工作的关注。目前只能支持1-2人，因为训练集没有收集更多人的图片。

When training a model on a single instance, does the batch size have to be set to 1? I tried setting the batch size to 2, but encountered dimension errors during the attention computation with query2. Can you explain this?

**Comment by gaoyixuan111** [comment]
> > 太棒了！请问还支持扩充到三人或四人吗？
> 
> 感谢您对我们工作的关注。目前只能支持1-2人，因为训练集没有收集更多人的图片。

When training a model on a single character, does the batch size have to be set to 1? I tried setting the batch size to 2, but encountered dimension errors during the attention computation with query2. Can you explain this?

**Comment by RedAIGC** [comment]
> > > 太棒了！请问还支持扩充到三人或四人吗？
> > 
> > 
> > 感谢您对我们工作的关注。目前只能支持1-2人，因为训练集没有收集更多人的图片。
> 
> When training a model on a single character, does the batch size have to be set to 1? I tried setting the batch size to 2, but encountered dimension errors during the attention computation with query2. Can you explain this?

Yes, currently only batch_size=1 is supported. If you need to increase the size, you need to modify the training code. You can try it.
